hw1/hw1_script.py

```diff
@@ -0,0 +1,84 @@
+# This function checks whether input sequence is valid. If so, decides whether it's DNA or RNA
+def check(seq):
+    if 'u' in seq.lower() and 't' in seq.lower():
+        return 'Invalid' #contains T and U simultaneously
+    elif seq.lower().replace('a','').replace('c','').replace('g','').replace('t','').replace('u','') != '':
+        return 'Invalid' #contains symbols besides nucleobases
+    elif 'u' in seq.lower():
+        return 'RNA' 
+    else:
+        return 'DNA'
+
+# This function returns transcribed sequence for DNA
+def transcribe(seq):
+    x = 'Tt'
+    y = 'Uu'
+    transcribe_table = seq.maketrans(x, y)
+    return seq.translate(transcribe_table)
+    
+# This function returns reversed sequence for DNA or RNA
+def reverse(seq):
+    return seq[::-1]
+
+# This function makes complementary sequence for DNA
+def dna_complement(seq):
+    x = 'ACGTacgt'
+    y = 'TGCAtgca'
+    complement_table = seq.maketrans(x, y)
+    return seq.translate(complement_table)
+
+# This function makes complementary sequence for RNA
+def rna_complement(seq):
+    x = 'ACGUacgu'
+    y = 'UGCAugca'
```

Comment:
Плохой нейминг переменных
```suggestion
    rna_nucl_ref = 'ACGUacgu'
    rna_nucl_comp = 'UGCAugca'
```